flashmob/go-guerrilla
# Client pool

To test:

Run the server.
Let one or more clients connect to it. Then send a kill signal:

    sudo kill `cat /var/run/go-guerrilla.pid

Tested the following scenarios:

1. kill with 2 clients connected
2. kill with no clients connected
3. kill while Borrow() is listening for pool close channel

Todo:

Server should respond with a shutdown message if killed in command state, otherwise in data state, let the client finish the message.
